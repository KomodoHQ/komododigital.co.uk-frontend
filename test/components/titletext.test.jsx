import React from 'react';
import TitleText from '../../src/components/titletext';
import renderer from 'react-test-renderer';

test('TitleText element is rendered', () => {
  const component = renderer.create(<TitleText />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
