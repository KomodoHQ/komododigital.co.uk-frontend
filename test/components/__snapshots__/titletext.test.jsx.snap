// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TitleText element is rendered 1`] = `
<div
  className="komodoGridWrapper title-wrapper   "
  style={
    Object {
      "background": "none",
    }
  }
>
  <div
    className="Title-Section "
  >
    <h2
      dangerouslySetInnerHTML={
        Object {
          "__html": "TITLE HERE",
        }
      }
    />
    <div />
    <div>
      <a
        className="showreel"
        href="https://vimeo.com/314800766/fef6932281"
        target="_blank"
      >
        Watch Showreel
      </a>
    </div>
  </div>
  <div
    className="image-wrapper "
  />
</div>
`;
